### PR TITLE
WEBEU2 - Preprocessing I - Josemaria Nriagu 🇳🇬

### PR DESCRIPTION
**Pull request by josenriagu**
Started working on project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
+
 <head>
   <meta charset="utf-8">
-
-  <title>My Resume</title>
-
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Resume | Josemaria Nriagu</title>
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
```
Comment (maximesalomon) [inline index.html]: Don't forget to add `<meta name="viewport" content="width=device-width, initial-scale=1.0">`

File: less/index.less [hunk 2]
```diff
@@ -4,56 +4,449 @@
...
+
+// set root element font-size
+
+:root {
+	font-size: 62.5%;
+}
+
+// LESS variables
+// define color schemes
+@primaryColor      : #E9ECEF;
```
Comment (maximesalomon) [inline less/index.less]: good use of variables!

Comment (maximesalomon) [reply less/index.less]: you could even create a separate LESS file for it and use @import

File: less/index.less [hunk 3]
```diff
@@ -4,56 +4,449 @@
...
+
+// define default section margins
+
+@bigsectiongap  : 4rem;
+@sectiongap     : 2rem;
+@smallsectiongap: 1rem;
+
+// define mixins for paragraphs and links
+
+.p-styles() {
```
Comment (maximesalomon) [inline less/index.less]: Nice use of mixins :)

**Comment by josenriagu** [comment]
Thanks a lot Max. I'll apply the corrections.

Regards,

Josemaria Nriagu
🇳🇬

**Pull request merged**